src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -2306,3 +2306,51 @@ function zonotope_vertices_fukuda_matrix(M::Union{MatElem,AbstractMatrix})
     Polymake.polytope.zonotope_vertices_fukuda(Oscar.homogenized_matrix(A, 1))
   )
 end
+
+@doc raw"""
+    vertex_figure(P::Polyhedron, n::Int; cutoff::Rational=1/2)
+
+Construct the vertex figure of the vertex `n` of a bounded polytope. The vertex figure is dual to a facet of the dual polytope. 
+
+# Optional Arguments
+-`cutoff::Number`: controls the exact location of the cutting hyperplane. It should lie in the open Interval $(0,1)$. 
+Value $0$ would let the hyperplane go through the chosen vertex, thus degenerating the vertex figure to a single point. 
+Value $1$ would let the hyperplane touch the nearest neighbor vertex of a polyhedron. Default value is $\fraction{1}{2}$. 
+-`no_coordinates::Bool`: Skip the coordinates computation, producing a pure combinatorial description. 
```

Comment:
Not sure how this will turn out in the online docs with the normal lines between the itemize. Maybe indenting will help, we should check this once the docs are rebuilt after adding the functions to the markdown files.
```suggestion
- `cutoff::Number`: controls the exact location of the cutting hyperplane. It should lie in the open Interval $(0,1)$. 
  Value $0$ would let the hyperplane go through the chosen vertex, thus degenerating the vertex figure to a single point. 
  Value $1$ would let the hyperplane touch the nearest neighbor vertex of a polyhedron. Default value is $\fraction{1}{2}$. 
- `no_coordinates::Bool`: Skip the coordinates computation, producing a pure combinatorial description. 
```